pkg/ottl/ottlfuncs/README.md

```diff
@@ -454,6 +455,20 @@ Examples:
 
 - `FNV("name")`
 
+### Hour
+
+`Hour(value)`
+
+The `Hour` Converter returns the hour from the specified time.
+
+`value` is a `time.Time`. If `value` is another type an error is returned.
+
+The returned type is `int64`.
```

Comment:
Please add a link to https://pkg.go.dev/time#Time.Hour somewhere, similar to how we link out for [truncateTime](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl/ottlfuncs#truncatetime)